templates/base.html

```diff
@@ -37,6 +37,17 @@
   <meta name="MobileOptimized" content="320">
   <meta name="viewport" content="initial-scale=1">
 
+  <!-- Open Graph -->
+  {% set image = post.config.image[0].src if post.config.image else '/static/images/blog/oddsite.jpg' %}
+  <meta property="og:image" content="{{ config.canonical_url }}static/images/{% block og_image %}{{ image }}{% endblock %}" />
+  <meta property="og:url" content="{{ config.canonical_url }}{% block og_url %}{% endblock %}" />
```

Comment:
Something inside the `og_url` block? Where will that data come from? Do we need the blocks around these?